DataEngineering/fetch_data.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+import pandas as pd
+import json
+import requests
+import mysql.connector as mysql
+
+url = "https://api.tfl.gov.uk/bikepoint" 
+response = requests.get(url)
+data = pd.DataFrame(json.loads(response.text))
+
+data['time'] = datetime.now()
+data.drop(['url','$type','placeType','children','childrenUrls'],axis=1,inplace=True)
+
+data['NbBikes'] = data['additionalProperties'].apply(lambda x : x[6]['value'])
```

Comment:
It would be better to iterate across this array and to search for the element which has NbBikes as key. It is not mentioned in the documentation that the element with key 'NbBikes' will always come as the 6th element in this array, so it might change in the future.